### DISCUSSION TITLE
$toast instance method is undefined

### DISCUSSION THREAD

**Discussion by sc-urs** [statamic/cms#6961] [Troubleshooting] [answered]
Hello, 

I have extended the control panel by adding additional .js file to resources folder which is then built inside the public folder (everything inside the file works as expected and is done according to documentation - https://statamic.dev/extending/control-panel#adding-css-and-js-assets)

https://statamic.dev/extending/toast-notifications

The problem I'm having is that the $toast instance method (`this.$toast.info('message');` as an example) is not available at all inside my javascript file.

Is there maybe a specific import or a step that is not included inside the documentation that I have to follow in order to get access to the method?

Thank you for your time!

**Comment by jasonvarga** [comment, marked as answer]
`this.$toast` will only be available to Vue components. If you're using it outside a Vue component you can do `Statamic.$toast`
